responsiveslider.js

```diff
@@ -0,0 +1,96 @@
+
+const slider = document.querySelector(`#slider`);
+const range = document.querySelector(`.range`);
+const nu = document.querySelector(`.nu`);
+const colorPickerJs = document.getElementById("color-picker");
+const submitButton = document.querySelector("input[type=submit]");
+const buttonSize = document.getElementById('buttonSize');
+const sliderSize = document.getElementById('sliderSize');
+const submit = document.getElementById('submitButton');
```

Comment:
"submit" might be a reserved word in html, pls check and rename your const.

Reply:
renamed submit to submitAll